src/styles/_pulse.scss

```diff
@@ -22,6 +24,28 @@
   animation: pulse 1.5s ease infinite;
 }
 
+:host([pulse][xxs]) {
```

Comment:
The auro-button is using the default size. The difference between default and xxs is not significant to support. 

I suggest removing this size spec from the Sass and the HTML demo.